dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -110,12 +177,17 @@ window.Dashboard = {
 				this.__handleGithubAuthChange(event.authenticated);
 			break;
 
+			case "CONTROLLER_UNREACHABLE_FROM_HTTPS":
+				// Controller isn't accessible via https. Redirect to http and try again.
+				window.location.href = window.location.href.replace("https", "http");
+			break;
+
+			case "HTTPS_CERT_MISSING":
+				Marbles.history.navigate("installcert");
+			break;
+
 			case "SERVICE_UNAVAILABLE":
-				if (window.location.protocol === "http:" && this.config.INSTALL_CERT) {
```

Comment:
@jvatic I removed this as you suggested. I'm not 100% sure that this does not affect some other issue. Please keep than in mind for the code inspection